op-node/rollup/derive/engine_update.go

```diff
@@ -150,6 +152,17 @@ func confirmPayload(
 	if err := sanityCheckPayload(payload); err != nil {
 		return nil, BlockInsertPayloadErr, err
 	}
+	if envelope.BlobsBundle != nil && len(envelope.BlobsBundle.Blobs) > 0 {
+		if updateSafe {
```

Comment:
Any explanation why `updateSafe` will return error?

Reply:
When `updateSafe`, it must be deriving instead of sequencing. Deriving is based on onchaindata and doesn't contain blob.

Reply:
I think it would be great to add the above comments here.

Reply:
Done.